modules/wordads/php/params.php

```diff
@@ -30,7 +31,7 @@ public function __construct() {
 				$option = $default;
 			}
 
-			$this->options[$setting] = (bool) $option;
+			$this->options[$setting] = $setting != 'wordads_custom_adstxt' ? (bool) $option : $option;
```

Comment:
yoda condition, you'll want here.
better to use `!==` as well, to keep the code sniffers happy